recipe/patches/0001-patch-redis-for-ar-ranlib.patch

```diff
@@ -1,5 +1,6 @@
 From 3ad7021f052e53cab2b6b01f6c6d08b86a5bbe76 Mon Sep 17 00:00:00 2001
-From: mattip <matti.picus@gmail.com>
+From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
+Co-Authored-By: Matti Picus <matti.picus@gmail.com>
```

Comment:
git will likely not be able to parse this correctly. Co-Authored-By lines go _after_ the `Subject:` (normally at the very end of the body of the commit message if there is one; the body must be separated from the title by an empty line).

Reply:
Thanks. I think it is OK to push this off for the next release, I will try to remember to properly redo the patches then.

Reply:
sure.